tools/gulpfile.js

```diff
@@ -134,39 +134,41 @@ gulp.task('update-react-native-svg', () => {
   });
 });
 
-gulp.task('update-react-native-gesture-handler', () => {
-  console.log(
-    'If you are updating Android, then also run update-react-native-gesture-handler-lib afterwards'
-  );
+gulp.task('update-react-native-gesture-handler-code', () => {
   return updateVendoredNativeModule({
-    argv,
-    name: 'react-native-gesture-handler',
-    repoUrl: 'https://github.com/expo/react-native-gesture-handler.git',
-    sourceIosPath: 'ios',
-    sourceAndroidPath: 'android/src/main/java/com/swmansion/gesturehandler/react',
-    targetIosPath: 'Api/Components/GestureHandler',
-    targetAndroidPath: 'modules/api/components/gesturehandler/react',
-    sourceAndroidPackage: 'com.swmansion.gesturehandler.react',
-    targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler.react',
-    installableInManagedApps: true,
+      argv,
+      name: 'react-native-gesture-handler',
+      repoUrl: 'https://github.com/expo/react-native-gesture-handler.git',
+      sourceIosPath: 'ios',
+      sourceAndroidPath: 'android/src/main/java/com/swmansion/gesturehandler/react',
+      targetIosPath: 'Api/Components/GestureHandler',
+      targetAndroidPath: 'modules/api/components/gesturehandler/react',
+      sourceAndroidPackage: 'com.swmansion.gesturehandler.react',
+      targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler.react',
+      installableInManagedApps: true,
+    });
   });
-});
-
-gulp.task('update-react-native-gesture-handler-lib', () => {
-  return updateVendoredNativeModule({
-    argv,
-    name: 'react-native-gesture-handler',
-    repoUrl: 'https://github.com/expo/react-native-gesture-handler.git',
-    sourceIosPath: 'ios',
-    targetIosPath: '',
-    sourceAndroidPath: 'android/lib/src/main/java/com/swmansion/gesturehandler',
-    targetAndroidPath: 'modules/api/components/gesturehandler',
-    sourceAndroidPackage: 'com.swmansion.gesturehandler',
-    targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
-    installableInManagedApps: true,
-    skipCleanup: true,
```

Comment:
Just to be on the safe side, are you sure that removing `skipCleanup` is ok here?

Reply:
I've checked it once more to be sure, and it works on both platforms. I can't assure it will in future, but at least it works now, which was not the case without those changes :)

Reply:
Seems I made a mistake while testing it. When I realized it, I made one more test and it failed. So I needed to apply some corrections, please take a look.